src/aura/autocomplete/autocomplete.cmp

```diff
@@ -77,6 +77,7 @@
                                   class="{!'slds-input ' + (and(v.showPills, !empty(v.value), !empty(v.displayValue)) ? 'slds-hide' : '')}"
                                   placeholder="{!v.placeholder}"
                                   value="{!v.displayValue}"
+                                  updateOn="keyup"
```

Comment:
well that is interesting. not sure why they would lock down the 'input' event.  but in any case, keyup is different than input.  keyup works if the user is using a keyboard to enter text.  it may or may not work if the user pastes text or text is entered in any other way.  have you tested pasting text?